src/manual/peer2peer/js/main.js

```diff
@@ -860,28 +873,40 @@ function doGetUserMedia_(constraints) {
       // Show the video element if we did request video in the getUserMedia call.
       var videoElement = $('local-view');
       videoElement.srcObject = stream;
+      registerVideoTrackEvents(stream);
       window.addEventListener('loadedmetadata', function() {
-          displayVideoSize(videoElement);}, true);
-      // Throw an error when no video is sent from camera but gUM returns OK.
-      stream.getVideoTracks()[0].onended = function() {
-        error_(global.localStream + ' getUserMedia successful but ' +
-               'MediaStreamTrack.onended event fired, no frames from camera.');
-      };
-      // Print information on track going to mute or back from it.
-      stream.getVideoTracks()[0].onmute = function() {
-        error_(global.localStream + ' MediaStreamTrack.onmute event has ' +
-            'fired, no frames to the track.');
-      };
-      stream.getVideoTracks()[0].onunmute = function() {
-        warning_(global.localStream + ' MediaStreamTrack.onunmute event has ' +
-                 'fired.');
-      };
+        displayVideoSize(videoElement);}, true);
     }
   }).catch(function(error) {
     error_('GetUserMedia failed with error: ' + error.name);
   });
 }
 
+function registerVideoTrackEvents(stream) {
```

Comment:
The whole page is constructed on this assumption and there is no functionality in the page to actually request a 2nd track (you would need to clone a track and add it to the mediastream)